analyzers/tests/SonarAnalyzer.UnitTest/TestFramework/Tests/VerifierTest.cs

```diff
@@ -153,34 +153,48 @@ public class Sample
                     """))
                 .Invoking(x => x.Verify()).Should().Throw<UnexpectedDiagnosticException>();
 
+        [TestMethod]
+        public void Verify_RazorWithAdditionalLocation()
+        {
+            using var scope = new EnvironmentVariableScope(false) { EnableRazorAnalysis = true };
+            DummyWithLocation.AddPaths("Dummy.SecondaryLocation.razor").Verify();
+        }
+
+        [TestMethod]
+        public void Verify_CshtmlWithAdditionalLocation()
+        {
+            using var scope = new EnvironmentVariableScope(false) { EnableRazorAnalysis = true };
+            DummyWithLocation.AddPaths("Dummy.SecondaryLocation.cshtml").Verify();
+        }
```

Comment:
These can be parametrized tests. I will merge it as is. Since I will do the same cleanup in a later PR for the other tests in this file and will handle this one too.